BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationHelper.m

```diff
@@ -120,6 +122,25 @@ + (UIWindow *)getApplicationKeyWindow {
     return [UIApplication sharedApplication].keyWindow;
 }
 
++ (CGSize)getApplicationWindowSize:(UIWindow *)window {
+    if (window) {
+        return window.bounds.size;
+    } else if ([self getApplicationKeyWindow]) {
+        return [self getApplicationKeyWindow].bounds.size;
+    } else {
+        return [[UIScreen mainScreen] bounds].size;
+    }
+}
+
++ (UIEdgeInsets)getApplicationWindowSafeAreaInsets:(UIWindow*)window API_AVAILABLE(ios(11.0)) {
+    if (window) {
+        return window.safeAreaInsets;
+    } else if ([BlueShiftInAppNotificationHelper getApplicationKeyWindow]) {
+        return [BlueShiftInAppNotificationHelper getApplicationKeyWindow].safeAreaInsets;
+    }
+    return  UIEdgeInsetsZero;
```

Comment:
shall we put this inside `else` block? just for improving the readability. Like the above method

Reply:
Yes, I can change it.